changelogs/fragments/9898-keycloak_user-supports-subgroups.yaml

```
@@ -0,0 +1,4 @@
+minor_changes:
+  - keycloak_user - user groups can now be referenced by their name, e.g. "staff", or their path, e.g.
+    "/staff/engineering".
+    The path syntax allows users to reference subgroups, which is not possible otherwise.
```

```suggestion
minor_changes:
  - keycloak module_utils - user groups can now be referenced by their name, like ``staff``, or their path, like
    ``staff/engineering``. This enables the module ``community.general.keycloak_user`` to make use of that feature.
    The path syntax allows users to reference subgroups, which is not possible otherwise (https://github.com/ansible-collections/community.general/pull/9898).
```

Please refer to the standard formatting for the changelog fragments' entries:

https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to-format

The outcome of what you write will be published in the collection changelog:

https://docs.ansible.com/ansible/latest/collections/community/general/changelog.html